ci-operator/step-registry/openshift/image-based/upgrade/seed/gather/lca/openshift-image-based-upgrade-seed-gather-lca-commands.sh

```
@@ -2,6 +2,7 @@
 set -euo pipefail
 
 trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+trap 'rc=$?; if [[ "${rc}" -ne 0 ]]; then echo "ERROR: Step failed at line ${LINENO} with exit code ${rc}"; fi' EXIT
```

Shellcheck (0.11.0)

[warning] rc is referenced but not assigned.

(SC2154)